LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright [2019-2023] [Bird-house community]
```

Comment:
Actually, I don't believe we should leave the square brackets
```suggestion
   Copyright 2019-2023 Bird-house community
```

Reply:
Hehe, I feel so noob with this error !